src/components/collection/Summary.tsx

```diff
@@ -0,0 +1,179 @@
+import truncate from 'lodash.truncate';
+
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Typography } from '@mui/material';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+
+import { Category, CategoryType, ItemCategory } from '@graasp/sdk';
+import { ItemRecord, MemberRecord } from '@graasp/sdk/dist/frontend/types';
+import { LIBRARY } from '@graasp/translations';
+
+import {
+  CATEGORY_TYPES,
+  ITEM_TYPES,
+  MAX_COLLECTION_NAME_LENGTH,
+} from '../../config/constants';
+import { compare } from '../../utils/helpers';
+import { QueryClientContext } from '../QueryClientContext';
+import ItemBreadcrumb from './ItemBreadcrumb';
+import Items from './Items';
+import SummaryDetails from './SummaryDetails';
+import SummaryHeader from './SummaryHeader';
+
+export const getParentsIdsFromPath = (
+  path: string,
+  { ignoreSelf = false } = {},
+) => {
+  if (!path) {
+    return [];
+  }
+
+  let p = path;
+  // ignore self item in path
+  if (ignoreSelf) {
+    // split path in half parents / self
+    // eslint-disable-next-line no-useless-escape
+    const els = path.split(/\.[^\.]*$/);
+    // if els has only one element, the item has no parent
+    if (els.length <= 1) {
+      return [];
+    }
+    [p] = els;
+  }
+  const ids = p.replace(/_/g, '-').split('.');
+  return ids;
+};
```

Comment:
Do we have some utility helper in sdk for this? @spaenleh

Reply:
Yes there is [the `getParentFromPath`](https://github.com/graasp/graasp-sdk/blob/48f3cb51367d4f2479eedb5791fd794894fafe86/src/utils/item.ts#L22) although not a drop-in-deplacement.

Reply:
I took this code from builder, but it looks like it is duplicated in each repo: [Builder](https://github.com/graasp/graasp-builder/blob/d0244adc48001feec062e687c70fff288e2b2788/src/utils/item.ts#L24) [Admin](https://github.com/graasp/graasp-admin/blob/d00585d744498e8c70179ad074eb5b462d5061f0/src/utils/item.js#L13) [Player](https://github.com/graasp/graasp-player/blob/725f17e1405421c3b9f88f09cb0ffa7dd7eddb83/src/utils/item.ts#L15) [UI](https://github.com/graasp/graasp-ui/blob/321b28daca8c7de7a9383e47524daf991d18893d/src/utils/utils.ts#L4)

I can import it from `graasp-query-client`:
```ts
import { getParentsIdsFromPath } from '@graasp/query-client/dist/utils/item'
```